### PR TITLE
docs: fix links in api docs

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Review by kara — APPROVED** [review]
LGTM

File: packages/common/src/location/platform_location.ts [hunk 1]
```diff
@@ -14,22 +14,18 @@ import {InjectionToken} from '@angular/core';
  * `PlatformLocation` encapsulates all calls to DOM apis, which allows the Router to be platform
  * agnostic.
  * This means that we can have different implementation of `PlatformLocation` for the different
- * platforms
- * that angular supports. For example, the default `PlatformLocation` is {@link
- * BrowserPlatformLocation},
- * however when you run your app in a WebWorker you use {@link WebWorkerPlatformLocation}.
+ * platforms that angular supports. For example, `@angular/platform-browser` provides an
+ * implementation specific to the browser environment, while `@angular/platform-browser` provides
```
Comment (gkalpak) [inline packages/common/src/location/platform_location.ts]: _platform-browser_ --> _platform-webworker_

File: packages/common/src/location/platform_location.ts [hunk 2]
```diff
@@ -14,22 +14,18 @@ import {InjectionToken} from '@angular/core';
...
+ * one suitable for use from with web workers.
```
Comment (gkalpak) [inline packages/common/src/location/platform_location.ts]: _from with_ --> _from_ or _with_

File: packages/core/src/application_ref.ts [hunk 3]
```diff
@@ -162,8 +162,8 @@ export function getPlatform(): PlatformRef|null {
  * has exactly one platform, and services (such as reflection) which are common
  * to every Angular application running on the page are bound in its scope.
  *
- * A page's platform is initialized implicitly when {@link bootstrap}() is called, or
- * explicitly by calling {@link createPlatform}().
+ * A page's platform is initialized implicitly when a platform is created via a platform factory
+ * (e.g. {@link platformBrowser}), or explicitly by calling {@link createPlatform} function.
```
Comment (gkalpak) [inline packages/core/src/application_ref.ts]: _calling_ --> _calling the_ (?)

**Comment by gkalpak** [comment]
No preview 😞

File: packages/core/src/metadata/view.ts [hunk 4]
```diff
@@ -9,14 +9,14 @@
...
- * See {@link ViewMetadata#encapsulation}.
+ * See {@link Component#encapsulation}.
  * @stable
  */
 export enum ViewEncapsulation {
   /**
    * Emulate `Native` scoping of styles by adding an attribute containing surrogate id to the Host
    * Element and pre-processing the style rules provided via
-   * {@link ViewMetadata#styles} or {@link ViewMetadata#stylesUrls}, and adding the new Host Element
+   * {@link Component#styles} or {@link Component#stylesUrls}, and adding the new Host Element
```
Comment (gkalpak) [inline packages/core/src/metadata/view.ts]: _stylesUrls_ --> _styleUrls_

File: packages/forms/src/validators.ts [hunk 5]
```diff
@@ -24,9 +24,6 @@ function isEmptyInputValue(value: any): boolean {
  *
  * Provide this using `multi: true` to add validators.
  *
- * ### Example
- *
- * {@example core/forms/ts/ng_validators/ng_validators.ts region='ng_validators'}
```
Comment (CSchulz) [inline packages/forms/src/validators.ts]: Is there no example any more?

Comment (IgorMinar) [inline packages/forms/src/validators.ts]: @kara I couldn't find this example. Do you know what happened to it? Did it ever exist?

File: packages/common/src/location/platform_location.ts (hunk 1 above)
Comment (IgorMinar) [inline packages/common/src/location/platform_location.ts]: thanks!

File: packages/common/src/location/platform_location.ts (hunk 2 above)
Comment (IgorMinar) [inline packages/common/src/location/platform_location.ts]: fixed

File: packages/core/src/application_ref.ts (hunk 3 above)
Comment (IgorMinar) [inline packages/core/src/application_ref.ts]: fixed

File: packages/core/src/metadata/view.ts (hunk 4 above)
Comment (IgorMinar) [inline packages/core/src/metadata/view.ts]: wow.. @petebacondarwin, how come dgeni didn't complain about this?

Comment (petebacondarwin) [inline packages/core/src/metadata/view.ts]: The reason is that these bits of content are not actually rendered on the page - so strictly they are not dangling links :-) - which indicates that we need to improve the template for "enum" pages.

Comment (IgorMinar) [inline packages/core/src/metadata/view.ts]: Shouldn't we error if we come across a link that is not rendered anywhere?

**Review by tbosch — APPROVED** [review]

File: packages/core/src/metadata/view.ts (hunk 4 above)
Comment (petebacondarwin) [inline packages/core/src/metadata/view.ts]: It is a bit tricky to ascertain but if you want to create an issue for it I'll look into it.

**Review by vicb — APPROVED** [review]

**Comment by CSchulz** [comment]
@petebacondarwin What about the missing examples https://github.com/angular/angular/pull/16326#discussion_r113394605 ?

**Comment by petebacondarwin** [comment]
This PR was to remove broken links. The example doesn't exist and so we have removed a link to a non-existent resource. The lack of the example does not block this PR, the rendered docs have not changed, since there was never an example being rendered in the first place.

If you want to create a new issue to create that example, that would be fine by me;

File: packages/forms/src/validators.ts (hunk 5 above)
Comment (CSchulz) [inline packages/forms/src/validators.ts]: It has been removed with 7606c96c80d3f1cb8161704e9646d23007345f1c.

Comment (petebacondarwin) [inline packages/forms/src/validators.ts]: Thanks for following up on this @CSchulz

Comment (CSchulz) [inline packages/forms/src/validators.ts]: I think it makes no sense to reuse that example. Perhaps we can copy the usage part of the Validator interface.

https://github.com/angular/angular/blob/master/packages/forms/src/directives/validators.ts#L25-L33

**Pull request merged**